edge/README.md

```diff
@@ -0,0 +1,21 @@
+# Maximo Visual Inspection Edge
+
+This folder hierarchy contains resources for Maximo Visual Inspection Edge (MVIE).
+
+MVIE is designed to bring the power of AI models trained with MVI to the edge as a self-contained solution that can run on edge nodes_ anything from servers to laptops to devices such as an nVidia Jetson or a Raspberry Pi.
+
+Edge nodes with GPUs can run models locally. For devices without GPUs, Edge can send inference requests to an MVI server or run the models on the CPU if lower performance is tolerable.
+
+
+The standard MVIE product install includes:
+- A Web UI and REST API, database, and MQTT Broker, all of which run in the `vision-edge-controller` container
+- A runtime engine to run inspection pipelines that acquire images (from cameras, folders of images, or video files), processes acquired images using the models, and runs the inferenence data through Rules which can be configured through the UI. The runtime is provided by the `vision-edge-cme` container
```

Comment:
'inferenence' mispelled

Reply:
are the rules required?
"old edge" could just run a model for inference only. What if someone is looking for that capability?

Reply:
> are the rules required?
> "old edge" could just run a model for inference only. What if someone is looking for that capability?

For inference only they do not use the runtime engine, which is the CME. Even with the CME, Rules are not required. IE an inspection can be run with no Rules created and that acts a lot like auto-labelling in MVI.